### PR TITLE
feat: add camera change on mouse scrollwheel

### LINKED ISSUE #1770: Mouse scroll forward / backward should switch camera mode
The only way to switch the camera mode right now is using the V shortcut. Scroll forward / backward should alternate between 1st and 3rd person camera, in preparation for future camera improvements.

**Acceptance Criteria**
- [ ] Mouse scroll forward switches to 1st person camera
- [ ] Mouse scroll backward switches to 3st person camera
- [ ] This only works on mouse hover in-world or mouse locked

### PR DESCRIPTION
**Pull request by DavideJensen**
## What does this PR change?

Fix #1770
Adds first/third person camera change on mousewheel 

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/mouse-scrollwheel-to-change-view
2. Use mouse scrollwheel to verify that the camera changes from first to third person and vice versa

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs [hunk 1]
```diff
@@ -69,6 +73,7 @@ private void Awake()
             }
 
             cameraChangeAction.OnTriggered += OnCameraChangeAction;
+            mouseWheelAction.OnValueChanged += OnMouseWheelChangeValue;
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs:76]: We should unsubscribe this on the `OnDestroy` method.

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]
I'm testing the change and the mouse wheel scrolling is not doing anything (I'm in Windows with a normal mouse).

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs (hunk 1 above)
Comment (DavideJensen) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/CameraController.cs:76]: Done!

**Comment by DavideJensen** [comment]
> I'm testing the change and the mouse wheel scrolling is not doing anything (I'm in Windows with a normal mouse).

That is really strange, I'll test now on my windows

**Review by sandrade-dcl — APPROVED** [review]
Now it is working perfectly for me, thanks! Good work!

**Pull request merged**